client/components/suggestions/docs/example.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SearchCard from 'components/search-card';
+import Suggestions from 'components/suggestions';
+
+class SuggestionsExample extends Component {
+
+	static displayName = 'Suggestions';
+
+	static hints = [ "Foo", "Bar", "Baz" ];
```

Comment:
This should be single quotes here. I guess the linter doesn't run in devdocs? Strange.
Both of these statics could actually be moved outside the component as constants. In fact, `displayName` could also be simplified to:

`SuggestionsExample.displayName = 'Suggestions';`